AirCasting/SessionViews/SessionLoadingView.swift

```diff
@@ -0,0 +1,29 @@
+// Created by Lunar on 24/06/2021.
+//
+
+import SwiftUI
+
+struct SessionLoadingView: View {
+    var body: some View {
+        HStack {
+            Image("ABLoading")
+                .frame(width: 60, height: 60)
+            VStack(alignment: .leading) {
+                Text("Your AirBeam is gathering data.")
+                    .font(Font.moderate(size: 15))
+                Text("Measurements will appear in 3 minutes.")
+            }
+            .font(Font.moderate(size: 13))
+            .lineSpacing(5)
```

Comment:
I would place it into HStack(spacing: 5) to make it one liner. Only my weird point of seeing 😄

Reply:
It's not a HStack, it's VStack of two texts. The lineSpacing is for situations where on the smaller phones (like mine) the text has more than one line :)